.circleci/config.yml

```diff
@@ -67,7 +68,11 @@ workflows:
           filters:
             tags:
               only: /.*/
-      - test-render-macos-chrome-dev:
+      - test-render-mac-chrome-dev:
```

Comment:
A reminder to address after merging: renaming will need to properly update `release-*` + `main` branch protection settings in https://github.com/mapbox/mapbox-gl-js/settings/branches.